runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -211,6 +226,17 @@ void doSerializeInternal(ByteBuf buf) {
         }
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This class overrides "equals()" and should therefore also override "hashCode()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1206)